fda-model/compute/simulation.cs

```diff
@@ -63,18 +63,9 @@ internal Simulation()
         public Results Compute(interfaces.IProvideRandomNumbers randomProvider, ConvergenceCriteria convergenceCriteria, bool computeDefaultThreshold = true, bool giveMeADamageFrequency = false)
         {
             //Validate();
-            if (HasErrors)
+            if (!CanCompute())
```

Comment:
didn't you say this is not clean code to negate a boolean in an if statement?  I'm ok with it, but wanted to make sure you were following the clean code guidelines you are interested in.

Reply:
you got me! I did read that this is not considered clean code. apparently in the weeds I thought otherwise. what do you think?